week1/3-website/css/style.css

```diff
@@ -1,8 +1,192 @@
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- */
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  background-color: #aa8d39;
+  width: 100%;
+  height: 2100px;
+  margin: 50px 50px 0px, 150px;
+  padding: 5px 0px 5px;
+  border-color: magenta;
+}
+
+.topic {
+  font-size: 2em;
+  text-align: center;
+  color:white;
+  background-color: #d4b16a;
+}
+
+/*the links for the menu*/
+.nav {
+  color: white;
+  background-color: rgb(197, 145, 40);
+  margin-bottom: 0px;
+}
+
+#navigation {
+  text-align: center;
+  position: flexible;
+  top: 0;
+  font-size: 25px;
+  padding: 0px;
+  margin-bottom: 0px;
+  }
+
+.link {
+  color: rgb(77, 13, 52);
+  text-align: center;
+  margin: 60px;
+}
+
+/*targets the heading child tag*/
+.pre > h2 {
+  text-decoration: underline;
+  margin: 60px;
+}
+
+.profile-image {
+  width: 150px;
+  height: 150px;
+  float:left;
+  border-radius: 10px;
+  border: 5px solid rgb(133, 79, 79);
+  margin: 30px;
+  height:auto;
+ }
+
+ /*group class code for logos*/
+ .solo, .codepen, .codeorg,  .capgemini {
```

Comment:
If the same CSS applies to 4 different classes, consider using a single class instead. See also line 112

Reply:
I have now assigned a class="course" to all of them and styled as  [.course].